ShoppingCartApp/src/main/java/com/example/SpringApp/Service/ItemService.java

```diff
@@ -0,0 +1,12 @@
+package com.example.SpringApp.Service;
+
+import com.example.SpringApp.Entity.Item;
+import com.example.SpringApp.Entity.User;
+
+import java.util.List;
+
+public interface ItemService {
+    public List<Item> getItemsByUser(int theId);
```

Comment:
Methods in an interface are by default public. So need not use public explicitly